src/main/java/no/digipost/signature/client/SignatureClient.java

```diff
@@ -17,15 +17,17 @@
 
 import no.digipost.signature.client.asice.CreateASiCE;
 import no.digipost.signature.client.asice.DocumentBundle;
-import no.digipost.signature.client.domain.Sender;
-import no.digipost.signature.client.domain.SignatureJob;
+import no.digipost.signature.client.domain.*;
 import no.digipost.signature.client.internal.CreateSignatureJobRequest;
 import no.digipost.signature.client.internal.SenderFacade;
-import no.digipost.signering.schema.v1.signature_job.SignatureJobRequest;
-import no.digipost.signering.schema.v1.signature_job.SignatureJobResponse;
+import no.digipost.signering.schema.v1.signature_job.XMLSignatureJobRequest;
+import no.digipost.signering.schema.v1.signature_job.XMLSignatureJobResponse;
+import no.digipost.signering.schema.v1.signature_job.XMLSignatureJobStatusResponse;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.InputStream;
+
 public class SignatureClient {
 
     private static final Logger LOG = LoggerFactory.getLogger(SignatureClient.class);
```

Comment:
Good idea. I've changed my IDEA-template after this was commited, so this won't happen again ;)